dash_html.py

```diff
@@ -124,64 +237,65 @@ def generate_control_card() -> html.Div:
     )
 
 
-def problem_details(solver: str) -> html.Div:
-    """generate the problem details section.
+def generate_problem_details_table(table_rows: tuple[dict]) -> html.Tbody:
+    """Generate the problem details table.
+
+    Args:
+        table_rows: A tuple of dicts where each dict represents a row in the table.
+
+    Returns:
+        html.Tbody: Tbody containing table rows for problem details.
+    """
+    rows = []
+    for table_row in table_rows:
+        row = []
+        for label, cell in table_row.items():
+            row += [label, cell]
+        rows.append(row)
+
+    return (
+        html.Tbody(
+            children=[html.Tr([html.Td(cell) for cell in row]) for row in rows],
+        ),
+    )
```

Comment:
Does this need to be returned as a tuple?

Reply:
`black` did this and I have no idea why

Reply:
updated https://github.com/dwave-examples/flow-shop-scheduling/pull/5/commits/6c9e8ca64844a1f71962ed15d87350bde9b8ee47

Reply:
I suspect the comma might have been there before (making it a tuple) with black just adding the parentheses , although I might be wrong. Black can be a mystery.